.travis.yml

```diff
@@ -63,7 +63,7 @@ env:
  global:
   - BUILD_FLAGS="-DCMAKE_BUILD_TYPE=Release"
   # don't use too many threads - may crash
-  - MAKEFLAGS="-j 2"
+  - MAKEFLAGS="-j 1"
```

Comment:
I don't mind, but is this intentional?

Reply:
yes, I use 1 thread when I debug, otherwise I don't understand what's gone wrong. will put it back to 2